recipes/raylib/all/conanfile.py

```diff
@@ -17,13 +17,17 @@ class RaylibConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "use_external_glfw": [True, False],
+        "opengl_version":[None, "3.3","2.1","1.1","ES 2.0"]
```

Comment:
```suggestion
        "opengl_version":[None, "3.3","2.1","1.1","ES-2.0"]
```
I would suggest not using white space, otherwise, you need to pass quoted string on CLI.